src/gg.mli

```diff
@@ -3127,13 +3127,13 @@ module Color : sig
       a color value. *)
 
   val v_srgb : ?a:float -> float -> float -> float -> color
-  (** [v r g b ~a] is the sRGB color [(r, g, b, a)] converted to a color
-      value. *)
+  (** [v r g b ~a] is the {e non-linear} sRGB color [(r, g, b, a)]
+      converted to a (linear) [Gg] color value. *)
```

Comment:
I'm not very keen on all these `non-linear` and `linear` you added. sRGB is by definition non-linear so it's more confusing than enlighting to say non-linear sRGB.